### DISCUSSION TITLE
Make @zwave-js/config paths "bundle friendly"

### DISCUSSION THREAD

**Discussion by robertsLando** [zwave-js/zwave-js#6556] [Feature Requests]
### Description

When playing with esbuild on zui I had to do some monkey patch on final bundle in order to make it work. To prevent this we could make the paths more bundle friendly using `require.resolve` 

### Do you have a solution in mind?

Store somewhere in a const `const baseDir = dirname(require.resolve('@zwave-js/config/package.json'))` and use that as base in `configDir` and `getEmbeddedConfigVersion` paths. Notice this will work only when zwave-js is used inside a module

### Describe alternatives you've considered

keep using the monkey patch

**Comment by AlCalzone** [comment]
step 1: https://github.com/zwave-js/node-zwave-js/pull/7343
step 2: https://github.com/zwave-js/node-zwave-js/pull/7344
step 3: https://github.com/zwave-js/node-zwave-js/pull/7345

**Reply by AlCalzone** [reply]
Fixed in 14.0.0-beta.1
